apps/web/src/utils/upload.ts

```
@@ -63,6 +63,15 @@ function parseSetEntry(entry: string): Set<string> {
   return set;
 }
 
+function nonVisibleCharChecker(entry: string | undefined) {
+  if (!entry) {
+    return null;
+  }
+  const nonVisibleCharCheck = /[^\u0009\u000A\u000D\u0020-\u007E\u00C0-\u024F]+/g.exec(entry);
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)